sql/moz-fx-data-shared-prod/search_derived/desktop_mobile_search_clients_monthly_v1/metadata.yaml

```diff
@@ -10,11 +10,18 @@ labels:
 scheduling:
   dag_name: bqetl_desktop_mobile_search_monthly
   referenced_tables: []
-  date_partition_parameter: submission_month
+  date_partition_parameter: null
+  destination_table:
+    >-
+    desktop_mobile_search_clients_monthly_v1${{
+    (execution_date +
+    macros.dateutil.relativedelta.relativedelta(months=-1,
+    day=1)).strftime("%Y%m%d")
+    }}
   parameters: ["submission_date:DATE:{{ds}}"]
 bigquery:
   time_partitioning:
     field: submission_month
-    type: month
+    type: day
```

Comment:
If you prefer to keep monthly partitions then the above `destination_table` would change to:

```
desktop_mobile_search_clients_monthly_v1${{
    (execution_date +
    macros.dateutil.relativedelta.relativedelta(months=-1,
    day=1)).strftime("%Y%m")
    }}
```

(removed the %d from the partition)